### PR TITLE
PPF-564 Add hold state with admin information

### PR DESCRIPTION
**Pull request by LucWollants**
### Added
- Added a new eloquent model `AdminEloquentModel` with a hold state and additional informational text
- Added simple Nova resource `AdminInformation` to edit the state and informational text and linked it to an `Integration`
- Added filter for the new hold state

---
Ticket: https://jira.uitdatabank.be/browse/PPF-564

### CODE REVIEW

File: app/Nova/Filters/AdminInformationFilter.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+namespace App\Nova\Filters;
+
+use Illuminate\Database\Eloquent\Builder;
+use Laravel\Nova\Filters\Filter;
+use Laravel\Nova\Http\Requests\NovaRequest;
+
+final class AdminInformationFilter extends Filter
+{
+    public $name = 'On hold';
```
Comment (grubolsch) [inline app/Nova/Filters/AdminInformationFilter.php:13]: Can you use constants for these values? Especially the internal values "on_hold" and "not_on_hold".

File: app/Nova/Filters/AdminInformationFilter.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            });
+        }
+
+        return $query;
+    }
+
+    public function options(NovaRequest $request): array
+    {
+        return [
+            'On Hold' => 'on_hold',
```
Comment (grubolsch) [inline app/Nova/Filters/AdminInformationFilter.php:35]: Capitalisation seems weird here? I would just use "On hold" and "Not on hold".

File: database/migrations/2024_11_21_165129_create_admin_information_table.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    public function up(): void
+    {
+        Schema::create('admin_information', static function (Blueprint $table) {
+            $table->uuid('id')->primary();
+            $table->uuid('integration_id');
```
Comment (grubolsch) [inline database/migrations/2024_11_21_165129_create_admin_information_table.php:14]: Couldn't integration_id use an index?

**Review by grubolsch — CHANGES_REQUESTED** [review]

File: app/Nova/Filters/AdminInformationFilter.php (hunk 1 above)
Comment (LucWollants) [reply app/Nova/Filters/AdminInformationFilter.php:13]: Here I fully agree that using a private constant does help 😊

**Comment by LucWollants** [comment]
@grubolsch I processed your feedback

**Review by grubolsch — APPROVED** [review]

File: app/Nova/Filters/AdminInformationFilter.php [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            });
+        }
+
+        return $query;
+    }
+
+    public function options(NovaRequest $request): array
+    {
+        return [
+            'On hold' => 'self::ON_HOLD',
```
Comment (grubolsch) [inline app/Nova/Filters/AdminInformationFilter.php:38]: 'On hold' => self::ON_HOLD,

**Review by grubolsch — CHANGES_REQUESTED** [review]

File: app/Nova/Filters/AdminInformationFilter.php (hunk 4 above)
Comment (LucWollants) [reply app/Nova/Filters/AdminInformationFilter.php]: Nicely spotted, fixed it.

**Review by grubolsch — APPROVED** [review]

**Pull request merged**